app/decorators/comment_decorator.rb

```diff
@@ -12,11 +12,12 @@ def render_with_children(commentable, all_comments)
     buttons = [h.link_to("Reply", h.new_comment_path(commentable_id: commentable.id, commentable_type: commentable_type(commentable), parent_id: model.id), class: "comment-button btn btn-default btn-xs"),
                h.link_to("Edit", h.edit_comment_path(model, commentable_id: commentable.id, commentable_type: commentable_type(commentable)), class: "comment-button btn btn-default btn-xs"),
                h.link_to("Delete", h.comment_path(model, commentable_id: commentable.id, commentable_type: commentable_type(commentable)), method: :delete, data: {confirm: "Sure?"}, class: "comment-button btn btn-default btn-xs")].join
+    author = h.content_tag(:dev, "By " + model.user.name, class: "comment-author")
```

Comment:
`h.content_tag(:div` ?

Think a `h.content_tag(:span` might work better since it'll let the items be in the same row.

Reply:
[nit] How about `"- " + + model.user.name`?